intranet/apps/eighth/tasks.py

```diff
@@ -66,6 +66,49 @@ def room_changed_single_email(
     )
 
 
+@shared_task
+def transferred_activity_email(
+    dest_act: EighthScheduledActivity, duplicate_students: Collection[get_user_model()]  # pylint: disable=unsubscriptable-object
```

Comment:
```suggestion
    dest_act: EighthScheduledActivity, duplicate_students  # pylint: disable=unsubscriptable-object
```
Type annotations generally aren't supposed to contain function calls.

In the past, I've just omitted type annotations on parameters that reference `User` in `eighth/models.py` 
-- `eighth/models.py` depends on `users/models.py`, so importing it would create circular import issues. I've since learned that it's possible to work around the circular import issue, but that would probably be a matter for another PR.